.circleci/config.yml

```diff
@@ -93,15 +46,78 @@ anchor_7: &setup_bazel_remote_execution
       sudo bash -c "echo -e 'build --config=remote' >> .bazelrc.user";
       echo "Reading from remote cache for bazel remote jobs.";
 
+# Executor Definitions
+executors:
```

Comment:
This and commands are pretty cool. I was wondering about multi-command abstractions a while back.